src/language/__tests__/blockString-fuzz.js

```diff
@@ -66,3 +70,77 @@ describe('printBlockString', () => {
     }
   }).timeout(20000);
 });
+
+describe('isPrintableBlockString', () => {
+  it('correctly tells if random strings are printable block string', () => {
+    // Testing with length >7 is taking exponentially more time. However it is
+    // highly recommended to test with increased limit if you make any change.
+    for (const fuzzStr of genFuzzStrings({
+      allowedChars: ['\n', '\t', ' ', '"', 'a', '\\'],
+      maxLength: 7,
+    })) {
+      if (isPrintableBlockString(fuzzStr)) {
+        const testStr = '"""' + fuzzStr + '"""';
+
+        let testValue;
+        try {
+          testValue = lexValue(testStr);
+        } catch (e) {
+          continue; // skip invalid values
+        }
+        invariant(typeof testValue === 'string');
+
+        const printedValue = lexValue(printBlockString(testValue));
+
+        invariant(
+          testValue === printedValue,
+          dedent`
+          Expected lexValue(printBlockString(${inspectStr(testValue)}))
+            to equal ${inspectStr(testValue)}
+            but got  ${inspectStr(printedValue)}
+        `,
+        );
+
+        const printedMultilineString = lexValue(
+          printBlockString(testValue, ' ', true),
+        );
+
+        invariant(
+          testValue === printedMultilineString,
+          dedent`
+          Expected lexValue(printBlockString(${inspectStr(
+            testValue,
+          )}, ' ', true))
+            to equal ${inspectStr(testValue)}
+            but got  ${inspectStr(printedMultilineString)}
+        `,
+        );
+      } else {
+        if (isBlank(fuzzStr)) {
+          continue;
+        }
+        const printedValue = lexValue(printBlockString(fuzzStr));
+
+        invariant(
+          fuzzStr !== printedValue,
+          dedent`
+          Expected lexValue(printBlockString(${inspectStr(fuzzStr)}))
+            not to equal ${inspectStr(fuzzStr)}
+        `,
+        );
+
+        const printedMultilineString = lexValue(
```

Comment:
In this test no need to test `printBlockString(fuzzStr, ' ', true)` it will slow down fuzzing
`printBlockString` fuzz tests already ensure that `lexValue(printBlockString(fuzzStr)) === lexValue( printBlockString(fuzzStr, ' ', true))` so no need to test it twice.